dataland-community-manager/src/test/kotlin/org/dataland/datalandcommunitymanager/email/BulkDataRequestMessageBuilderTest.kt

```diff
@@ -0,0 +1,112 @@
+    }
+
+    private fun buildInternalBulkEmailMessageMock() {
+        Mockito.`when`(
+            cloudEventMessageHandlerMock.buildCEMessageAndSendToQueue(
+                ArgumentMatchers.anyString(),
+                ArgumentMatchers.anyString(),
+                ArgumentMatchers.anyString(),
+                ArgumentMatchers.anyString(),
+                ArgumentMatchers.anyString(),
+            ),
+        ).then() {
+            val arg1 = objectMapper.readValue(it.getArgument<String>(0), InternalEmailMessage::class.java)
+            val arg2 = it.getArgument<String>(1)
+            val arg3 = it.getArgument<String>(2)
+            val arg4 = it.getArgument<String>(3)
+            val arg5 = it.getArgument<String>(4)
+
+            Assertions.assertEquals("Dataland Bulk Data Request", arg1.subject)
+            Assertions.assertEquals("A bulk data request has been submitted", arg1.textTitle)
+            Assertions.assertEquals("Bulk Data Request", arg1.htmlTitle)
+            Assertions.assertEquals(authenticationMock.userDescription, arg1.properties.getValue("User"))
+            Assertions.assertEquals("2020, 2023", arg1.properties.getValue("Reporting Periods"))
+            Assertions.assertEquals(
+                bulkDataRequest.dataTypes.joinToString(", ") { it.value },
+                arg1.properties.getValue("Requested Frameworks"),
+            )
+            Assertions.assertEquals(
+                acceptedCompanyIdentifiers.joinToString(", "),
+                arg1.properties.getValue("Accepted Companies (Dataland ID)"),
+            )
+            Assertions.assertEquals(MessageType.SendInternalEmail, arg2)
+            Assertions.assertEquals(correlationId, arg3)
+            Assertions.assertEquals(ExchangeName.SendEmail, arg4)
+            Assertions.assertEquals(RoutingKeyNames.internalEmail, arg5)
+        }
+    }
+
+    @Test
+    fun `validate that the output of the bulk internal email message sender is correctly build`() {
+        buildInternalBulkEmailMessageMock()
+        val properties = mapOf(
+            "User" to authenticationMock.userDescription,
+            "Reporting Periods" to formatReportingPeriods(bulkDataRequest.reportingPeriods),
+            "Requested Frameworks" to bulkDataRequest.dataTypes.joinToString(", ") { it.value },
+            "Accepted Companies (Dataland ID)" to acceptedCompanyIdentifiers.joinToString(", "),
+        )
+        val message = InternalEmailMessage(
+            "Dataland Bulk Data Request",
+            "A bulk data request has been submitted",
+            "Bulk Data Request",
+            properties,
+        )
+        cloudEventMessageHandlerMock.buildCEMessageAndSendToQueue(
+            objectMapper.writeValueAsString(message),
+            MessageType.SendInternalEmail,
+            correlationId,
+            ExchangeName.SendEmail,
+            RoutingKeyNames.internalEmail,
+        )
+    }
```

Comment:
Dieser gesmte Test macht so wie er gerade ist keinen Sinn, da er keinerlei "realen" Code abtestet, sondern den Mock selbst.
Was eigentlich hier getestet werden sollte, ist die Funktion `sendBulkDataRequestInternalMessage` so wie ich das mit Florian in einem Call erörtert habe.